pkg/controller/performanceprofile/components/profile/profile.go

```diff
@@ -10,6 +10,10 @@ import (
 const (
 	hugepagesSize2M = "2M"
 	hugepagesSize1G = "1G"
+	topologyPolicyNone = "none"
```

Comment:
Sory, missed it, there are constants already available for these values.
See https://github.com/openshift-kni/performance-addon-operators/blob/master/vendor/k8s.io/kubelet/config/v1beta1/types.go#L59
and example usage in our functests: https://github.com/openshift-kni/performance-addon-operators/blob/master/functests/performance/topology_manager.go#L47